packages/backend/types/index.ts

```
@@ -1,4 +1,5 @@
 import { z } from "zod";
+import { z } from "zod";
```

Biome (2.1.2)

[error] Shouldn't redeclare 'z'. Consider to delete it or rename it.

'z' is defined here:


(lint/suspicious/noRedeclare)